docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Asynchronous webhooks
+---
+
+## Architecture
+
+Asynchronous webhooks can be used for reacting to the events emitted by the Saleor. Worker instance will send event payload via the chosen protocol. Your app should acknowledge that payload was received. The result of app operations is not sent back to the Saleor. If you need to send data back to the user (for example, calculated shipping cost), you should use [synchronous webhook](developer/extending/apps/synchronous-webhooks.mdx).
+
+## Example use cases
+
+- send order confirmation to company chat.
+- synchronize stocks with an external warehousing system.
+- update Elasticsearch or Algolia index after product description change.
+- trigger frontend SSR for new pages.
+
+## Permissions
+
+Managing webhooks is available for users with `MANAGE_APPS` permission. The app can also create and modify its own webhooks without the need for additional permissions.
+
+## Webhook protocols
+
+Webhooks support below protocols:
+
+- HTTP(S) 
+- Google Cloud Pub/Sub
+- AWS SQS
+
+Protocol is chosen based on `targetUrl` scheme.
+
+### HTTP(S)
+
+Webhook payloads are sent in POST requests.
+
+All webhooks with `targetUrl`, where the scheme is HTTP(S) will use this protocol.
+
+Saleor calculates the HMAC sha256 header - `X-Saleor-Signature` based on `secretKey` and the payload.
+
+If success response is not received, it will retry its request after a delay time in case of temporary service unavailability. 
+
+
+### Google Cloud Pub/Sub
```

Comment:
Let's move both queues above HTTP(S) as they are a better approach and mention that they don't support synchronous callbacks.

Reply:
Synchronous callback is a separate page, so theres no confusion here. 
IMO https as first is okay because most of devs who start creating apps is more familiarized with webhooks than queue.

But adding suggestion note is good idea. How about:
"Using queue/pub sub has advantage of better monitoring and fault handling"?

Reply:
The main disadvantage of HTTP(S) is that Saleor Cloud will hug your endpoint to death if there are more events than you can handle. It's also possible to lose data if your endpoint goes down for long enough for all retries to fail.

Reply:
Added additional guidance info with queue recommendation